ansible/Dockerfile

```diff
@@ -1,23 +1,25 @@
 FROM alpine:3.6
 
 ARG CI_HELPER_VERSION
+ARG GLIBC_VERSION
+ARG PYTHON_VERSION
 
 WORKDIR /root
 RUN apk update &&\
  apk add -q --no-cache 
-RUN apk add bash 
-RUN apk add ca-certificates 
-RUN apk add curl 
-RUN apk add git 
-RUN apk add openssh-client 
-RUN apk add python 
-RUN apk add py-pip 
-RUN apk add ansible
-RUN echo "Install CI Helper" && \
-    curl -sSL https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/sgerrand.rsa.pub -o /etc/apk/keys/sgerrand.rsa.pub && \
-    curl -sSL https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/glibc-2.23-r3.apk -O && \
-    apk add -q glibc-2.23-r3.apk && \
-    rm /etc/apk/keys/sgerrand.rsa.pub glibc-2.23-r3.apk 
-RUN curl -sSL https://github.com/rande/gitlab-ci-helper/releases/download/${CI_HELPER_VERSION}/linux-amd64-gitlab-ci-helper -o /usr/bin/ci-helper && \
+RUN apk add bash \
```

Comment:
It should be merged with the previous `RUN`, in which `apk add` does nothing by itself without a package name anyway ?